react-frontend/src/components/StyleComponents/card.js

```diff
@@ -225,6 +225,16 @@ export const CommunityCardHeader = styled(CardClickableHeader).attrs({
   padding: 24px;
 `;
 
+export const CommunityCardHeading = styled.h5.attrs({
+  className: 'communityCardHeading',
+})`
+  background: #003366;
+  color: #ffffff;
+  font-weight: 700;
+  font-size: 21px;
+  padding: 24px;
+`;
```

Comment:
This slightly messed up the Community card styling:

![Screen Shot 2021-09-23 at 9 01 45 AM](https://user-images.githubusercontent.com/16946297/134544598-f0d1a633-17e0-4c91-a6bf-f3427d59cb45.png)

This can easily be fixed by removing the `padding` and also adding `margin-bottom: 0;`:

![Screen Shot 2021-09-23 at 9 14 23 AM](https://user-images.githubusercontent.com/16946297/134544814-765b39ea-0c8c-4686-bcb7-a1c40cf68c0f.png)

I clicked around to all the other pages with cards and didn't notice anything else amiss.